system/ee/ExpressionEngine/Service/Thumbnail/Thumbnail.php

```diff
@@ -68,34 +68,34 @@ public function __construct(File\FileSystemEntity $file = null)
             } elseif ($file->isEditableImage() || $file->isSVG()) {
                 $this->url = $file->getAbsoluteThumbnailURL();
                 $this->path = $file->getAbsoluteThumbnailPath();
-                $this->tag = '<img src="' . $this->url . '" alt="' . $file->title . '" title="' . $file->title .'" class="thumbnail_img" />';
+                $this->tag = '<img src="' . $this->url . '" alt="' . $file->description . '" title="' . $file->title .'" class="thumbnail_img" />';
```

Comment:
I think many people would use file description for longer description, so we need to run it through `ee()->functions->word_limiter` and make sure HTML is stripped off, if we want to use it